fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -151,7 +153,7 @@ const CreateElection = () => {
   // Confirm button only clickable when the Name, Question and 2 Ballot options have values
   const confirmButtonEnabled: boolean =
     isConnected === true &&
-    electionName !== '' &&
+    electionName.trim() !== '' &&
```

Comment:
Same comment as above. Try to separate the value displayed by the user and the one used for processing and perform the check on the one used for processing.

At the moment you perform the processing and the checks in the same place.